examples/tools/scene_viewer/camera_controller_plugin.rs

```diff
@@ -0,0 +1,174 @@
+//! A freecam-style camera controller plugin.
+//! To use:
+//! - Load `CameraControllerPlugin` into your App
```

Comment:
```suggestion
//! - Copy the code for the `CameraControllerPlugin` and add it to your App
```

Reply:
Modified both plugin doc comments to be more explicit regarding common use case of end-users, as per suggestion.